src/Proxy.hs

```diff
@@ -1793,7 +1857,7 @@ putGTUDropR addrText = do
             atrPayload = encodePayload payload
             atrHeader = TransactionHeader {
               thSender = dropAccount,
-              thNonce = nonce,
+              thNonce = nanNonce nonce, -- VH/FIXME: This is only reliable if all account transactions are finalized. Is this OK?
```

Comment:
This is only reliable if all account transactions are finalized. Is this OK?

Reply:
@abizjak @Bargsteen this says outdated, but is still relevant. I do not know if you can see this and other comments I made.

Reply:
We should keep it as it was. There is retry logic in the wallet-proxy that should handle cases like this.

We cannot do anything completely reliable unless we keep the nonce locally, which adds other complexities.